ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -707,7 +728,7 @@ then
 		fi
 	fi
 
-	if ver hwcmp PX4FMU_V2 PX4FMU_V4 PX4FMU_V4PRO MINDPX_V2 PX4FMU_V5
+	if ver hwcmp PX4FMU_V2 PX4FMU_V4 PX4FMU_V4PRO MINDPX_V2 PX4FMU_V5 OMNIBUS_F4SD
```

Comment:
Is this on purpose? Since the px4flow driver is not enabled in the cmake file.

Reply:
ahh, yes. thx. brought back px4flow in the cmake config